service/src/main/resources/application.yml

```diff
@@ -38,10 +38,12 @@ spring:
   liquibase:
     change-log: changelog/changelog.yaml
 
+# This is the hostname of Adam's laptop and is NOT generalizable
+# Workaround for unexplained inability to use `http://localhost:8000`
 workflow-engines:
   cromwell:
-    baseUri: "http://localhost:8000"
-    healthUri: "http://localhost:8000/engine/v1/status"
+    baseUri: "http://wm334-2ce.lan:8000"
+    healthUri: "http://wm334-2ce.lan:8000/engine/v1/status"
```

Comment:
Created ticket to fix my setup and this weirdness: https://broadworkbench.atlassian.net/browse/BW-1311

Reply:
Fixed!